spec/formatador_spec.rb

```diff
@@ -42,7 +42,7 @@ def all
     end
 
     it "returns formatted representation" do
-      Fog::Formatador.format(@collection).must_equal @expected
+      _(Fog::Formatador.format(@collection)).must_equal @expected
```

Comment:
I'm not familiar with this `_()` invocation, could you explain it to me?

Reply:
This is minitest and the old style is deprecated. https://minitest.rubystyle.guide/#global-expectations mentions this and https://docs.rubocop.org/rubocop-minitest/cops_minitest.html#minitestglobalexpectations can automatically replace them for you.

The alternative is to rewrite everything to `assert_equal()` & friends instead.

Reply:
K, thanks. I hadn't bumped into that previously.